app/components/pages-menu/component.js

```diff
@@ -4,6 +4,20 @@ export default Ember.Component.extend({
     style: Ember.computed('layer.settings.values.color', function(){
         return Ember.String.htmlSafe('color: ' + (this.get('layer.settings.values.color') || '#333') + '; ');
     }),
+    sticky: Ember.observer('layer.settings.values.StickToTop' , function() {
+        let topOfNav = $('.pages-menu').offset().top;
+        if(this.get('layer.settings.values.StickToTop')){
+            $(window).on('scroll', function (e) { 
```

Comment:
Do you think it makes sense to add this somewhere here? https://emberjs.com/api/ember/2.14/namespaces/Ember.run/methods/debounce?anchor=debounce